### ISSUE TITLE
strings to translate, not equal to sentences

### ISSUE THREAD

**Issue by althio** [hotosm/learnosm#502]
I don't know how the NL people managed to translate, but the sentences in Transifex are awfully cut where there is newlines in the raw English files... and that is very different from a sentence, a paragraph or a logical clause.

Is it possible to establish guidelines for the English files to be less messy?

**Comment by Nick-Tallguy** [comment]
Hi @althio I think this is mostly because of the way the site has evolved. Many of the sections started life as documents, possibly on google. They were adapted by a process outlined in the wiki, but it led to many of the modules being formatted with every item being a bullet point. When I've carried out some of the amendments I've tried to reduce the number of bullet points and changed the formatting - it's something that needs continuous work to improve. 

There's actually very little which I think is 'new' - mainly the coordination section.

I agree that it must make the translation difficult. 

The site is so extensive that it needs a team continually working on it, to carry out a number of needed improvements. It's mainly firefighting at the moment , where it would be better if there were enough people working on it so we could actually plan & manage.
